### ISSUE TITLE
Update TooManyRequestsException to reflect new behavior of Moneybird API

### ISSUE THREAD

**Issue by holtkamp** [picqer/moneybird-php-client#200]
**Context**
Moneybird has updated the [throttling](https://developer.moneybird.com/#throttling) mechanism. 

- `Retry-After` now indicates the (absolute) timestamp _after which_ a retry can be done, while this library [interprets](https://github.com/picqer/moneybird-php-client/blob/e02cff1acc9aff7063de6c17e64c3db45e39432b/src/Picqer/Financials/Moneybird/Connection.php#L540-L543) it as _the (relative) number of seconds it_ should wait

**Problem**
So functionally the behavior has changed. Now something like this occurs:

```
 API request limit reached: sleep the indicated amount of 1586950800 seconds
```

Which causes a sleep time of 50 years, instead of a sleep time up until `2020-04-15T11:40:00+00:00`

**Solution**
We should probably update the [`TooManyRequestsException`](https://github.com/picqer/moneybird-php-client/blob/master/src/Picqer/Financials/Moneybird/Exceptions/Api/TooManyRequestsException.php) to reflect this and add support for the new headers:

- `RateLimit-Remaining` containing the time remaining before a new request can be made.
- `RateLimit-Limit` containing the total limit.
- `RateLimit-Reset` containing the time after which a new request can be made.

**Comment by gldrenthe89** [comment]
Any update on this?

**Comment by holtkamp** [comment]
@gldrenthe89 I changed my own code to deal with the new "behavior" properly and added a "TODO"... Coming months I probably won't have time to come up with a PR for this library, maybe you can give it a try?

**Comment by gldrenthe89** [comment]
created a [PR](https://github.com/picqer/moneybird-php-client/pull/203)
This is just a simple solution to get it back to how it worked. 
the header `RateLimit-Remaining` is just the seconds to wait before you can try again. Just like before with the `Retry-After` header

**Comment by gldrenthe89** [comment]
Added 2 additional commits to the [PR](https://github.com/picqer/moneybird-php-client/pull/203) set 2 new variables that are accessible in the exception 
`$currentRateLimit`
 and 
`$rateLimitResetsAfterTimestamp`

**Comment by stephangroen** [comment]
Changes of @gldrenthe89 have been merged in his bigger PR: #230. Thanks @gldrenthe89 and sorry for the delay.

**Comment by gldrenthe89** [comment]
> Changes of @gldrenthe89 have been merged in his bigger PR: #230. Thanks @gldrenthe89 and sorry for the delay.

Not a problem. Happy that it is merged now ;-)
